styles.css

```diff
@@ -0,0 +1,209 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+}
+
+.header-container {
+  background-image: url("images/header-image.jpg");
+  background-repeat: no-repeat;
+  display: flex;
+  background-color: grey;
+  width: 100%;
+  height: 30rem;
+  position: relative;
+}
+
+.header-logo {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin-left: 50px;
+  width: 20%;
+
+}
+
+.header-container p {
+  color: #81B741;
+  font-size: 1.2rem;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.cart-container {
+  display: flex;
+
+}
+
+
+.cart-icon {
+  display: flex;
+  width: 2rem;
+  position: absolute;
+  top: 5%;
+  right: 10%;
+  color: black;
+
+
+}
+
+.checkout-button {
+  width: 2rem;
+  position: absolute;
+  top: 15%;
+  right: 15%;
+  padding: 1rem;
+  font-size: bold;
+}
+
+
+.nav {
+  position: relative;
+  background-color: grey;
+  width: 100%;
+  height: 3rem;
+}
+
+  .nav ul li  {
+   list-style: none;
+   display: inline-block;
+   margin-left: 5%;
+   float: left;
+   color: white;
+   padding: 1rem;
+
+
+ }
+
+ li:after {
+   content: "/";
+   padding: 1rem;
+
+ }
+
+ .adopt-section-container {
+   display: flex;
+   flex-direction: row;
+   justify-content: space-between;
+   align-items: flex-start;
+   flex-wrap: wrap;
+   margin-left: 8%;
+   margin-right: 8%;
+   margin-top: 5%;
+   position: relative;
+
+ }
+
+
+ .adopt-section-image {
+   display: flex;
+   padding: 1rem;
+
+ }
+
+ .adopt-section-text {
+   display: flex;
+   padding: 1rem;
+   width: 60%;
+   flex-wrap: wrap;
+ }
+
+ .adopt-section-text h2 {
+   display: block;
+   width: 100%;
+   align-items: center;
+   justify-content: center;
+
+ }
+
+ .main-container {
+   position: relative;
+   display: flex;
+   flex-direction: row;
+   justify-content: space-evenly;
+   align-items: flex-start;
+   flex-wrap: wrap;
+   height: auto;
+   margin-top: 5%;
+   margin-left: 8%;
+   margin-right: 8%;
+   margin-bottom: 5%;
+
+ }
+
+ .dog-box {
```

Comment:
Cool, is a great start to these layouts. I'm sure you'll fold in the rest of the content in soon, but the flex layout is smart